_posts/articles/2021-02-14-setting-up-techdocs-on-backstage.md

```diff
@@ -0,0 +1,210 @@
+---
+layout: post
+title: Setting up Techdocs on Backstage
+excerpt: "Techdocs is Spotify's homegrown docs like code solution. it allows the user to store documentation to near code thus allowing it to be easily discovered."
+last_modified_at: Sun Feb  12 14:39:05 CST 2021
+categories: articles
```

Comment:
I debated this point myself, but came out on the article side because there's some precedence set there and better findability evidence. Based on GA, a couple of the more popular articles on the site are https://www.docslikecode.com/articles/balsamiq-case-study-part-2/ and https://www.docslikecode.com/articles/api-docs-with-code/, probably because the keywords are super relevant. I can move it over though if you feel strongly about it.